README.md

```diff
@@ -89,28 +131,78 @@ space.on('membersUpdate', (members) => {
 ]
 ```
 
-2. When a member has left the space in one of the following ways:
-
-- called `space.leave()`
-- has closed the tab
-- abruptly disconnected from the internet for more than 2min (which is configurable via [`offlineTimeout`](https://github.com/ably-labs/spaces/blob/main/docs/class-definitions.md#offlinetimeout):
+The following is an example of the response sent by subscribing to `membersUpdate` when a member has left the space by calling `space.leave()`, closing the application or has been disconnected for less than 2 minutes (default `offlineTimeout` setting):
 
 ```json
 [
   {
     "clientId": "user1234",
     "isConnected": false,
+    "location": null,
     "lastEvent": {
-      "name": "enter",
-      "timestamp": 1677595689759
+      "name": "leave",
+      "timestamp": 1678890015000
     },
-    "location": null,
-    "profileData": {
-      "username": "Example User",
-      "avatar": "https://example.com/users/1234.png"
-    }
   }
 ]
 ```
 
-**Class definitions are available [in the docs folder](/docs/class-definitions.md).**
+#### Subscribe to locations
+
+Location data can be used to collect a variety of data which can be used to create UI elements representing a user's location. For example, showing a user's visual location on a page, their active URL, or anything else showing where a user is active in an application.
```

Comment:
```suggestion
Location data can represent a user's location within the collaborative application. This can be used to create contextual UI elements such as an active border around the selected cell in a spreadsheet. Other examples of location include showing a user's visual location on a page, their active URL, the page they are currently active on, etc.
```